packages/contracts-bedrock/scripts/upgrades/holocene/templates/bundle_template.json

```diff
@@ -0,0 +1,96 @@
+{
+  "chainId": $L1_CHAIN_ID,
+  "metadata": {
+    "name": "Holocene Hardfork - Contracts Upgrade",
+    "description": "Upgrades the `SystemConfig.sol`, `MIPS.sol`, `FaultDisputeGame.sol`, and `PermissionedDisputeGame.sol` contracts for Holocene."
+  },
+  "transactions": [
+    {
+      "metadata": {
+        "name": "Upgrade `SystemConfig` proxy",
+        "description": "Upgrades the `SystemConfig` proxy to the new implementation, featuring configurable EIP-1559 parameters."
+      },
+      "to": "$PROXY_ADMIN_ADDR",
+      "value": "0x0",
+      "data": "$TX_1_PAYLOAD",
+      "contractMethod": {
+        "type": "function",
+        "name": "upgrade",
+        "inputs": [
```

Comment:
Do we need to reinitialise this or is just upgrading sufficient?

Reply:
Just upgrading is sufficient. Holocene derivation code expects that the values will be zero to start out with - the first set will emit the `ConfigUpdate` event and bring the parameters in the contract in sync with the parameters placed in L2 header `extraData`.

Reply:
Cool, just wanted to check.